### ISSUE TITLE
Base Asset Price in BSD-Option-Price Excel spreadsheet

### ISSUE THREAD

**Issue by ybl84f1** [isamkov/ms-office-vba#1]
Greetings, thanks for your contributions.

I was using the BSD-Option-Price spreadsheet and noticed the two graphs "**Price and Delta (on base asset price)**" and "**Vega and Theta (on base asset price)**" were incorrectly using the Strike Price instead of the Base Asset Price. I believe the error is with cell N53 which gets it's value from cell D5 ("Strike") instead of cell D4 ("BA").

Also, I believe Rho in cells H9 and I9 is incorrectly calculated. In both cases the formula used:
 "=-Strike*T * EXP(-riskrate*T) * N__d2/10000"

should be:
"=-Strike*T * EXP(-riskrate*T) * N__d2/100"

in other words **_divide by 100 and not 10000_**.

Great tool - thanks for sharing.
